tests/app/views/handlers/test_view_submitted_response_pdf.py

```diff
@@ -33,4 +33,4 @@ def test_filename_uses_schema_name(storage, schema, language):
     questionnaire_store.set_metadata({"schema_name": "test_view_submitted_response"})
     pdf = ViewSubmittedResponsePDF(schema, questionnaire_store, language)
 
-    assert pdf.filename == "test_view_submitted_response.pdf"
+    assert pdf.filename == f"test_view_submitted_response-{str(date.today())}.pdf"
```

Comment:
Just checking the trello card, and I think we are supposed to be using the `schema title` rather than the `schema name` for the first part?